pom.xml

```diff
@@ -719,6 +743,20 @@
               </execution>
             </executions>
           </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-gpg-plugin</artifactId>
```

Comment:
Where are the signature files?

Reply:
I wanted to discuss this part.  I see two options, we can include the them in the repository, encrypted and reference them from the `/travis` folder, or we can include them at the Travis account level, and inject them as environment variables.

Here are links to the two approaches:

[Continuous Integration to maven central (for free)](https://www.phillip-kruger.com/post/continuous_integration_to_maven_central/) *this one uses environment variables at the account level*

[Complete guide to continuous deployment to maven central from Travis CI](http://www.debonair.io/post/maven-cd/) *this one uses encrypted files and values in the `travis.yml`*

I can see both working, but I would prefer not to have the values or encrypted files in the repository, if we can avoid it.

Reply:
I like `encrypted files and values in the travis.yml`